style.css

```diff
@@ -48,4 +49,151 @@ body {
 .text {
   color: #c8c8d4;
   font-size: 20px;
+  font-weight: 400;
+}
+
```

Comment:
- [ ]Please use a display grid for your main card container, it should be more flexible and allow you to implement desktop version

Reply:
@BenMukebo Sorry I don't get this part, I mean:
My card containers/pages can flex all to 1000px and beyond.

And I thought the project was about the mobile version, can you clear to me what do you mean by "allow you to implement desktop version". 

Politely:relaxed::relaxed:, why my version can't be expanded to desktop?
(Appreciate all the knowledge that you can give me with your answers, please be abundant, thanks:smile::smile:)